torq/umbrel-app.yml

```diff
@@ -46,26 +46,22 @@ gallery:
   - 3.jpg
   - 4.jpg
 releaseNotes: >-
-  New features:
-
-  * Users can sign and verify signatures
-
-  *  Vector monitoring is now ready (and users can sign using Torq to get started)
-
-  *  Channel point column in the channel stable
-
-  *  In app "Help chat"
-
-  Fixes:
-
-  *  Several UI fixes
-
-  *  Links to mempool, amboss and 1ml are working again
-
-  *  Fix events toggling on channel inspect page
+  This release includes:
+  
+  * Automation!
+  
+  Automation is finally here, or at least the first part of it. You can now automate the channel policy (fee rate etc.) and add or remove tags. We will very soon release more advanced actions and of course Rebalancing!
+  
+  * Detailed data import choices
+  
+  You can now select what data Torq import. This is an important feature for very large nodes that are concerned about the impact of the initial data import. You can gradually turn them on again.
+  
+  * No login required on Umbrel
```

Comment:
And an extra line above this?

Reply:
@nevets963 I've added the extra spacing requested, is there a way to preview the release notes?

Reply:
@laitang2000 Yes :) Through SSH, you can checkout your fork using `sudo ~/umbrel/scripts/repo checkout https://github.com/lncapital/umbrel-apps#torq-v0.18.17` and then that will use your fork as the basis for the app store